### ISSUE TITLE
Rendering board as ascii

### ISSUE THREAD

**Issue by vanniktech** [Netflix-Skunkworks/go-jira#195]
Ideally something like `jira board` which would yield some ascii generated table showing all the lines and where tickets are currently positioned.

**Comment by coryb** [comment]
Good idea, I agree this would be useful, however most of the apis dealing with sprint boards are pretty fussy to use.  I would find this command useful myself, so I may get around to it, but I am very limited on time.
